responses.py

```diff
@@ -640,9 +640,15 @@ def _on_request(self, adapter, request, **kwargs):
                 return _real_send(adapter, request, **kwargs)
 
             error_msg = (
-                "Connection refused by Responses: {0} {1} doesn't "
-                "match Responses Mock".format(request.method, request.url)
+                "Connection refused by Responses - the call doesn't "
```

Comment:
```suggestion
                u"Connection refused by Responses - the call doesn't "
```
Could hit a UnicodeError here in Python 2